src/components/AddressSearch/index.js

```diff
@@ -179,11 +184,11 @@ function AddressSearch({
             language: preferredLocale,
             types: resultTypes,
             components: isLimitedToUSA ? 'country:us' : undefined,
+            locationbias: locationBias || 'ipbias',
```

Comment:
I get your point. Instead of passing `ipbias`, it may be better to let the default behavior occur by not passing the `locationbias` property if we do not have one. We can do something like this below. 
Let me know what you think on this.

`...(locationBias && {locationbias: locationBias}),`